### PR TITLE
feat(): add texwork

### PR DESCRIPTION
**Pull request by gilgda**
Added the TeXwork tutorial

### CODE REVIEW

File: src/content/guides/TEXWORK/TEXWORK.MD [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+   ![Screenshot 1](content/guides/TEXWORK/latex1.png)
+
+3. Klicke nun auf die Kategorie Textsatz
+
+4. Klicke nun auf den grünen Knopf um einen Pfad hinzuzufügen!
+
+   ![Screenshot 2](content/guides/TEXWORK/latex2.png)
+
+5. Nun öffnet sich ein Fenster, klicke auf den Stift und gib folgenden Pfad ein und klicke auf öffnen /home/samba/progs/texlive/bin/i386-linux/
```
Comment (Lordnoname) [inline src/content/guides/TEXWORK/TEXWORK.MD]: Please wrap the path, using `''`

File: src/content/guides/TEXWORK/TEXWORK.MD [hunk 2]
```diff
@@ -0,0 +1,21 @@
+1. Öffne TeXWorks
+
+2. Klicke auf Bearbeiten und dann auf Einstellungen
+
+   ![Screenshot 1](content/guides/TEXWORK/latex1.png)
+
+3. Klicke nun auf die Kategorie Textsatz
+
+4. Klicke nun auf den grünen Knopf um einen Pfad hinzuzufügen!
```
Comment (Lordnoname) [inline src/content/guides/TEXWORK/TEXWORK.MD]: Please provide consistant punctuation marks

File: src/content/guides/LOESCHEN/LOESCHEN.MD [hunk 3]
```diff
@@ -0,0 +1,7 @@
+1. Wählen Sie die gewünschte Datei aus.
+
+2. Drücken Sie die Tasten >SHIFT< und >Entf<
```
Comment (Lordnoname) [inline src/content/guides/LOESCHEN/LOESCHEN.MD]: Please wrap this line, using `''`

File: src/content/guides/LOESCHEN/LOESCHEN.MD [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
+3. Klicken Sie auf Löschen.
```
Comment (Lordnoname) [inline src/content/guides/LOESCHEN/LOESCHEN.MD]: Same here

File: src/content/guides/TEXWORK/TEXWORK.MD [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+   ![Screenshot 2](content/guides/TEXWORK/latex2.png)
+
+5. Nun öffnet sich ein Fenster, klicke auf den Stift und gib folgenden Pfad ein und klicke auf öffnen /home/samba/progs/texlive/bin/i386-linux/
+
+   ![Screenshot 3](content/guides/TEXWORK/latex3.png)
+
+6. Klicke nun so oft wie möglich auf den grünen Pfeil um den Pfad nach oben zu verschieben!
```
Comment (Lordnoname) [inline src/content/guides/TEXWORK/TEXWORK.MD]: Same here

File: src/content/guides/LOESCHEN/LOESCHEN.MD [hunk 6]
```diff
@@ -0,0 +1,7 @@
+1. Wählen Sie die gewünschte Datei aus.
```
Comment (Lordnoname) [inline src/content/guides/LOESCHEN/LOESCHEN.MD]: Same as below

**Comment by Lordnoname** [comment]
@DevVersion please review

**Comment by DevVersion** [comment]
@Lordnoname LGTM

**Pull request closed without merging**